SS14.Launcher/HappyEyeballsHttp.cs

```diff
@@ -25,6 +26,27 @@ public static HttpClient CreateHttpClient(bool autoRedirect = true)
             AllowAutoRedirect = autoRedirect
         };
 
+        if (!string.IsNullOrEmpty(proxyURL))
+        {
+            WebProxy webProxy = new WebProxy(proxyURL);
+            Uri proxyURLUri = new Uri(proxyURL);
```

Comment:
What happens if the proxy settings are malformed (invalid URL)? Does it cause the launcher to get stuck crashing on startup with broken config?